### ISSUE TITLE
Not really adaptive graph

### ISSUE THREAD

**Issue by Scaramouch33** [kenziyuliu/DGNN-PyTorch#19]
Recently, I have go through the code and find that in this implementation, the number of the edges is still fixed. But if we input all the possible edges into the network of 9 layers, the memory will boom. I wonder whether there are some good ideas to use all possible edges and can stack many layers too.

**Comment by kenziyuliu** [comment]
Hi, 

I think by "adaptive" the paper means that the edge weights in the graph are learnable through backprop. This line allows gradients for the graph: https://github.com/kenziyuliu/DGNN-PyTorch/blob/master/model/dgnn.py#L64

**Comment by 15762260991** [comment]
> Hi,
> 
> I think by "adaptive" the paper means that the edge weights in the graph are learnable through backprop. This line allows gradients for the graph: https://github.com/kenziyuliu/DGNN-PyTorch/blob/master/model/dgnn.py#L64

@kenziyuliu Is the same principle of "adaptive" in MS-G3D? Looking forward to your reply！

**Comment by kenziyuliu** [comment]
Hi @15762260991 this line in the MS-G3D repo might be relevant: https://github.com/kenziyuliu/MS-G3D/blob/master/model/ms_gcn.py#L40

**Comment by 15762260991** [comment]
> Hi @15762260991 this line in the MS-G3D repo might be relevant: https://github.com/kenziyuliu/MS-G3D/blob/master/model/ms_gcn.py#L40

Thank you for your patience and guidance!
